aiida/orm/utils/load_funcs.py

```diff
@@ -0,0 +1,205 @@
+# -*- coding: utf-8 -*-
```

Comment:
Would rename this file `load_funcs.py` to `loaders.py`.

Reply:
Ha, so there is already a loader.py in that module. Open to other suggestions though

Reply:
Ah yeah of course, my bad. But perfect, that is where they should go. I would just put them at the top of that file.

Reply:
done 👍